src/common/utils/setup-helpers.ts

```diff
@@ -301,7 +301,7 @@ function setLogOptions(options: KalturaPlayerConfig): void {
     Utils.Object.createPropertyPath(options, 'log', {});
   }
 
-  if (options.log && (options.log as any).useDebugInfo) {
+  if (!options.log || (options.log as any).useDebugInfo !== false) {
```

Comment:
I think you can do only 1 that combines them:
`options.log?.useDebugInfo !== false`